File: Model/StayAccess.cs
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;

namespace Model
{
    [Table("stay_access")]
    public class StayAccess
    {
        [Key]
        [DatabaseGenerated(DatabaseGeneratedOption.Identity)]
        [Column("id")]
        public int? Id { set; get; }
        [Column("stay_id")]
        public int? StayId { set; get; }
        public Stay Stay { set; get; }

        [Column("label")]
        public string Label { set; get; }
    }
}

File: WebAPP/ClientApp/src/dto/stayAccess.ts
import { Stay } from "./stay";

export class StayAccess {
    public id: number;
    public stayId: number;
    public stay: Stay;
    public label: string;

    constructor(id?: number, stayId?: number, stay?: Stay, label?: string)

    constructor(id: number, stayId: number, stay: Stay, label: string) {
        this.id = id;
        this.stayId = stayId;
        this.stay = stay;
        this.label = label;
    }
}
